inc/template-functions.php

```diff
@@ -149,3 +149,30 @@ function nightingale_read_more_posts( $title, $link ) {
 
 }
 
+function nightingale_add_google_analytics () {
```

Comment:
Nothing wrong here but it would be great if we could define it in WP as a variable?